app/src/main/java/org/p2p/wallet/send/ui/dialogs/SendConfirmBottomSheet.kt

```diff
@@ -82,6 +93,37 @@ class SendConfirmBottomSheet(
         }
     }
 
+    private fun initPinView() = with(binding) {
+        pinView.apply {
+            setFingerprintVisible(true)
+            onPinCompleted = ::checkPinCode
+            onBiometricClicked = {
+                isVisible = false
+                confirmBiometrics()
+            }
+        }
+    }
+
+    private fun confirm() {
+        onConfirmed.invoke()
+        dismissAllowingStateLoss()
+    }
+
+    private fun checkPinCode(pinCode: String) {
+        CoroutineScope(dispatchers.io).launch {
+            when (authInteractor.signInByPinCode(pinCode)) {
```

Comment:
why we use interactor inside fragment ?

Reply:
I think we need to create presenters for BottomSheet now too

Reply:
`SendConfirmBottomSheet` is inherited from `NonDraggableBottomSheetDialogFragment`. There are also another bottom sheets inherited from that class and using interactors. I think we can add non draggable logic to `BaseMvpBottomSheet` and get rid of `NonDraggableBottomSheetDialogFragment` without presenters. But it looks like separate refactoring task, so let's do this outside this PR. What do you think, guys?

Reply:
https://p2pvalidator.atlassian.net/browse/PWN-3831